seedfarmer/commands/_deployment_commands.py

```diff
@@ -848,7 +848,15 @@ def apply(
 
     manifest_path = os.path.join(config.OPS_ROOT, deployment_manifest_path)
     with open(manifest_path) as manifest_file:
-        deployment_manifest = DeploymentManifest(**yaml.safe_load(manifest_file))
+        manifest_input = yaml.safe_load(manifest_file)
+    ## Override all regions with the local session region if local
+    if DeployModuleFactory.is_local():
```

Comment:
approving... this will work but I have an alternative idea that I have yet to test